app/javascript/styles/mastodon/components.scss

```diff
@@ -10489,7 +10482,6 @@ noscript {
 
   &__text {
     flex: 1 1 auto;
-    font-style: 14px;
```

Comment:
It's meant to be `font-size`

Reply:
I have restored it as `font-size: 14px` (both occurrences).